src/main/java/com/emc/pravega/perf/PerfStats.java

```diff
@@ -149,7 +149,8 @@ private synchronized void record(long event, int bytes, Instant startTime, Insta
         }
 
         if (this.printer != null) {
-            printer.printRecord(count, event, bytes, latency);
+            final long nanotime = startTime.getEpochSecond() * 1000000000L + startTime.getNano();
```

Comment:
Please, do this conversion using a constant for `1000000000L` if there is no method that already does that for you.